kolibri/deployment/default/wsgi.py

```diff
@@ -7,9 +7,15 @@
 https://docs.djangoproject.com/en/1.8/howto/deployment/wsgi/
 """
 import os
+import time
 
 from django.core.wsgi import get_wsgi_application
+from django.db.utils import OperationalError
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "kolibri.deployment.default.settings.base")
 
-application = get_wsgi_application()
+try:
+    application = get_wsgi_application()
+except OperationalError:  # It happens when sqlite vacuum is being executed. The db is locked
+    time.sleep(60)  # vacuum should not take longer than 1 minute
+    application = get_wsgi_application()  # try again one last time
```

Comment:
Waiting a whole minute might lead someone to believe things are broken if they are waiting around for it? 
 Maybe this should be a loop with some logging? e.g.

```python
application = None
tries_remaining = 6
interval = 10
while not application and tries_remaining:
    try:
        logging.info("Starting Kolibri")
        application = get_wsgi_application()
    except OperationalError:
        # an OperationalError happens when sqlite vacuum is being executed. the db is locked
        logging.info("DB busy. Trying again in {}s".format(interval))
        tries_remaining -= 1
        time.sleep(interval)
        application = get_wsgi_application()  # try again one last time
if not application:
    logging.error("Could not start Kolibri")
```

Reply:
yep, I like that idea.